models/user.go

```diff
@@ -1492,8 +1492,9 @@ func (opts *SearchUserOptions) toConds() builder.Cond {
 		cond = cond.And(keywordCond)
 	}
 
-	if !opts.Private {
-		// user not logged in and so they won't be allowed to see non-public orgs
+	if len(opts.Visible) > 0 {
+		cond = cond.And(builder.In("visibility", opts.Visible))
+	} else {
```

Comment:
I think to remove the `else` is better?

Reply:
what do you mean? else ... default value (show only public)

Reply:
As a consideration for a compitable with before, it's also OK.